app/routes/verkiezingen/verkiezingsuitslag/prepare.js

```diff
@@ -59,6 +59,7 @@ export default class PrepareInstallatievergaderingRoute extends Route {
         'bekleedt.bestuursfunctie',
         'heeft-lidmaatschap.binnen-fractie',
         'status',
+        'beleidsdomein',
```

Comment:
Why is this necessary?

Reply:
to decrease number of calls to resources